scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -459,14 +460,30 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                                     .replace(/\]/g, '')
                                     .split(',');
 
-                                for (var i = 0; i < modifiedErrors.length; i++) {
-                                    var message = _.trim(modifiedErrors[i]);
-                                    // the message format is 'Field error text' (contains ')
-                                    var field = message.split(' ')[0].substr(1);
+                                getLabelNameResolver().then((getLabelForFieldId) => {
```

Comment:
Why not include the field name in the JSON response from the server so string parsing is not required?